src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -540,14 +541,35 @@ public Platform getPlatformName() {
     }
 
     public String getFullAppPath() {
+
+        File dir = new File(appFile);
+        Path path = Paths.get(appFile);
```

Comment:
Will we run into any exceptions if we don't do the null check before trying to set the `path`?

Reply:
Good call. e428235